tests/api/v1/samples/getWithLimit.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
oh, yeah, i hadn't thought that through

for consistency, we *should* be implementing limit/offset on find samples too (esp. since this whole thing became an issue because of performance of GET /v1/samples)

i mean, it's still valuable at the db level for all other resources, but we need a solution for samples, too